my_sprites.py

```diff
@@ -542,6 +576,9 @@ def on_key_press(self, key, modifiers):
             self.atttack_pressed = True
             self.attack()
 
+        if previous_direction in [self]:
```

Comment:
What is the logic here? If the *previous_direction* (A Direction) is in a list with only _self_? That should never be true??